flyteplugins/go/tasks/plugins/webapi/agent/integration_test.go

```diff
@@ -40,22 +40,22 @@ type MockPlugin struct {
 type MockAsyncTask struct {
 }
 
-func (m *MockAsyncTask) GetTaskMetrics(ctx context.Context, in *admin.GetTaskMetricsRequest, opts ...grpc.CallOption) (*admin.GetTaskMetricsResponse, error) {
-	panic("not implemented")
+func (m *MockAsyncTask) GetTaskLogs(ctx context.Context, in *admin.GetTaskLogsRequest, opts ...grpc.CallOption) (service.AsyncAgentService_GetTaskLogsClient, error) {
```

Comment:
Can you add more descriptions about why the integration tests are changed?
And how can users use these functions  `GetTaskLogs` and `GetTaskMetrics`?

Reply:
The interface of rpc service is changed, so we should update it here. However, we should remove this mock client anyway, and use the [one](https://github.com/flyteorg/flyte/pull/4718/files#diff-f57ee6c74861be45abc9cdca16d517132ed19cfda1675a13bc1286ce5d69843bR16-R26) generated by mockgen.